OutOfSchool/OutOfSchool.DataAccess/Repository/IQuartzJobRepository.cs

```diff
@@ -0,0 +1,7 @@
+﻿using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository;
+
+public interface IQuartzJobRepository : IEntityRepository<long, QuartzJob>
```

Comment:
same as above, just inject `IEntityRepository<long, QuartzJob>` in constructor and it works.

Reply:
The interface and the repository were removed.